conduct/enforcement.md

```diff
@@ -7,20 +7,12 @@ and fair.
 ## The Code of Conduct Committee
 
 All responses to reports of conduct violations will be managed by a Code of
-Conduct Committee ("the committee"), currently comprised of
-[*note:* still gathering volunteers, this will be filled in as they become available and prior to final upload into the rendered site]:
-
-* Person One foo@email (Chair of the Committee)
-* ...
-* Person Five bar@email
-
-The Jupyter Steering Council will approve the members of this five person
-committee. The members will be drawn from our entire community, and we intend
-that it *not* be only Steering Council members: anyone who is actively engaged
-with Jupyter can propose to join this committee (we'll explicitly solicit
-volunteers when short of five members).  One member will be designated chair of
-the group and will be responsible for all reports back to the Steering Council.
-
+Conduct Committee ("the committee"). The Code of Conduct Committee is current
```

Comment:
"...is current**ly**" or "The current Code of Conduct Commity is the Jupyter Steering Council" or without passive voice "The Code of Conduct Committee consists of all members of the Jupyter Steering Council"

Reply:
Until the Code of Conduct Committee has been selected, all the members of the Jupyter Steering Council will serve as the interim Code of Conduct committee.

Reply:
I have addressed these comments and expanded the language to also cover situations where the CoC isn't available to handle an incident report promptly. Let me know what you think.